get-started/quickstart.rst

```diff
@@ -0,0 +1,395 @@
+:doc:`../scripting/index` section.
+
+Managing Zeek
+=============
+
+Zeek comes packaged with ZeekControl (``zeekctl``) to manage more complex
+deployments.
+
+The same network device used in the Zeek command line can be used with
+``zeekctl``. This will go in a configuration file. For the following example,
+``$PREFIX`` will refer to the installation directory. This is likely
+``/usr/local/zeek`` if built from source or ``/opt/zeek`` if installed from a
+pre-built package.
+
+First, update the configuration’s interface in ``$PREFIX/etc/node.cfg``. If
+the device is ``en0``, that would look like:
+
+.. code-block:: console
+
+     [zeek]
+     type=standalone
+     host=localhost
+     interface=en0
+
+You can further configure the ``local.zeek`` script found in
+``$PREFIX/share/zeek/site/local.zeek``. ``zeekctl`` loads this script by 
+default. It is not overwritten by Zeek upgrades.
+
+Run ``zeekctl`` in order to start an interactive prompt and manage your Zeek
+deployment:
+
+.. code-block:: console
+
+     $ zeekctl
+     Hint: Run the zeekctl "deploy" command to get started.
+
+     Welcome to ZeekControl 2.5.0-76
+
+     Type "help" for help.
+
+     [ZeekControl] >
+
+Then run ``deploy`` to get started:
+
+.. code-block:: console
+
+     [ZeekControl] > deploy
+
+In another terminal, run the same two curl commands from before:
+
+.. code-block:: console
+
+     $ curl -X GET http://zeek.org
+     $ curl -X WEIRD http://zeek.org
+
+Then return to the ZeekControl prompt and stop it:
+
+.. code-block:: console
+
+     [ZeekControl] > stop
+     stopping zeek ...
+     [ZeekControl] > exit
+
+And exit from ``zeekctl``:
+
+.. code-block:: console
+
+     [ZeekControl] > exit
+
+The logs from ZeekControl will not appear in your current directory. Instead,
+they will appear in ``$PREFIX/logs/current`` when running. Since the process was
+stopped, they will appear in a directory with the current date within 
+``$PREFIX/logs/`` - such as ``$PREFIX/logs/2025-01-01/``.
+
+These logs are compressed as ``.log.gz`` files from gzip_. You may decompress
+these via ``gunzip`` then read them, or use gzip’s packaged ``zcat`` command.
+On Mac, this looks like:
+
+.. code-block:: console
+
+     $ zcat < $PREFIX/logs/2025-01-08/weird.11:03:38-11:03:43.log.gz
+     <...>
+     1736352218.157077       CFvENWVlkwVHhLL35       2603:6081:18f0:99e0:7da2:6b81:9a83:cb4e 57823   2606:2800:21f:cb07:6820:80da:af6b:8b2c   80      unknown_HTTP_method     WEIRD   F       zeek    -
+
+The logs contain the ``WEIRD`` HTTP request.
+
+More information on using ZeekControl can be found in the
+`ZeekControl documentation`_. More information on setting up a cluster can be
+found in the :doc:`cluster-setup` section.
+
+Clusters
+========
+
+ZeekControl is also used to manage a cluster of Zeek processes. This cluster
+contains many processes which analyze traffic together. Clusters are normally
+split among many machines. For this example, all nodes will be local.
```

Comment:
> normally split among many machines

I'm not sure this is actually true. I think more people run a single host with multiple processes than multi-host clusters.

Reply:
I reworded it to not spread falsities :)